README.md

```diff
@@ -6,47 +6,59 @@ Here are a few use cases for this action:
 
 ## Applying a "triage" label on any new issue:
 
-```workflow
-workflow "Triage" {
-  on = "issue"
-  resolves = "Apply Triage Label"
-}
-
-action "Apply Triage Label" {
-  uses = "actions/github@v1.0.0"
-  args = "label triage --action=opened" # Only when issues are opened!
-  secrets = ["GITHUB_TOKEN"]
-}
+```yml
+name: Triage
+on: [issues]
+jobs:
+  applyTriageLabel:
+    name: Apply Triage Label
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Apply Triage Label
+        uses: actions/github@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          args: label triage --action=opened
```

Comment:
If we do that, we could remove `--action=opened` here I presume?

Reply:
Yes.